cached_network_image/example/pubspec.yaml

```diff
@@ -21,7 +21,7 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  flutter_lints: ^1.0.4
+  flutter_lints: ^2.0.1
```

Comment:
This is not really missed, I was planning to do this later.

Reply:
Okay. So leave it as this change? Basically no impact by this update though because new rules in updated packages are not used anyway. In a separate commit, I'd like to suggest using all_lint_rules_community to include new rules automatically.